tests/test_websocket_decoder.c

```diff
@@ -185,7 +185,8 @@ DECODER_TEST_CASE(websocket_decoder_rsv) {
         };
 
         /* Set the appropriate RSV */
-        input[0] |= (uint8_t)(1 << (6 - rsv));
+        /* the bit arithmetic is setup this way to avoid Conversion warnings from the compiler. */
+        input[0] = (uint8_t)(input[0] | (1 << (6 - rsv)));
```

Comment:
I wonder if it's a better idea to just disable the compiler warnings in places than rewrite bit-twiddling code so it's even harder to understand. Bit-twiddling code is hard enough to grok as it is.

Reply:
I strongly dislike the wConversion option as worthless due to its false positive rate. If you all want to remove the warning fine. But it doesn’t build as is.